src/kernel/capinv.c

```diff
@@ -127,9 +128,11 @@ kmem_unalloc(unsigned long *pte)
 	 */
 	unsigned long old = *pte;
 
-	assert(old & PGTBL_COSKMEM);
+	/* TODO: remove after test assert(chal_pgtbl_chal2cos(old) & PGTBL_COSKMEM); */
```

Comment:
/* TODO: */ here don't really convey the importance.
Why do you think it should be removed/kept should have been part of it. Applies to all /*TODO: remove after test xxxxxxxxxx */